server/config/forms/statementValidation.test.js

```diff
@@ -1,3 +1,8 @@
+jest.mock('moment', () => () => ({
```

Comment:
jest/babel previously hoisted this up to the beginning of the file - we need to do it manually now as using ts-jest rather than babel for compilation of tests